README.md

```diff
@@ -38,17 +38,17 @@ As a short summary, the recommendations are threefold:
 3.  Confirm REUSE compliance
 
 You are recommended to read the
-[recommendations](https://reuse.software/practices/) in full for more
+[our tutorial](https://reuse.software/tutorial) in full for more
 details.
```

Comment:
"for more details" is probably inaccurate here. I'd rephrase to:

> You are recommended to read our tutorial for a step-by-step guide through these three steps. For more details, you may read the specification.

With appropriate links etc etc etc

Reply:
See #91

Reply:
Thanks! I borrowed text from the REUSE 3.0 news item to link to tutorial, faq and spec